source/compression/OP_COMPRESSED.rst

```diff
@@ -194,7 +194,7 @@ Compressor IDs
 Each compressor is assigned a predefined compressor ID.
 
 +-----------------+----------------+-------------------------------------------------------+
-| compressorId    | isMaster Value |  Description                                          |
+| compressorId    | hello Value    |  Description                                          |
```

Comment:
At risk of a larger diff, would "handshake value" be more consistent with what you wrote above?

This would also allow an opportunity to remove the extra space before "noop" in the following row.